src/hevm/src/EVM/Solidity.hs

```diff
@@ -106,6 +107,11 @@ data SlotType
 --  | StorageArray AbiType
   deriving Eq
 
+data Code = Code
```

Comment:
Is it possible to unify the `Code` types between here and in `Dapp.hs`? Seems a little weird to have these two almost identical types hanging around.

Reply:
oops